packages/test/test-utils/src/opProcessingController.ts

```diff
@@ -163,18 +207,23 @@ class DeltaManagerMonitor extends DeltaManagerToggle {
             // to exclude any message that was sent before we start monitoring the delta manager
             this.firstClientSequenceNumber = messages[0].clientSequenceNumber;
         }
+        // if we are not active, the outbound with nack, and we will reconnect write
+        // this flag tracks the process. after reconnection, the op will be resubmitted
+        // on the write connection and reset this flag
+        this.pendingWriteConnection = !this.deltaManager.active;
         for (const message of messages) {
-            switch (message.type) {
-                case MessageType.NoOp:
-                    // TODO: server have some heuristic to delay or coalesce no-ops
-                    // Can't really track it for now
-                    break;
-                case MessageType.Summarize:
-                    this.pendingCount += 2; // expect SummaryAck too.
-                    break;
-                default:
-                    this.pendingCount++;
-                    break;
+            // TODO: server have some heuristic to delay or coalesce no-ops
+            // Can't really track it for now
+            if (message.type !== MessageType.NoOp) {
+                switch (message.type) {
+                    case MessageType.Summarize:
+                        this.pendingCount += 2; // expect SummaryAck too.
```

Comment:
is SummaryAck sent on behalf of this client? i.e. it's clientId property points to this client?
This assumes no nacks, right?
I'd rather count it as one, but ignore SummaryAck when inbounding and counting messages for given client - it feels cleaner to me.

Reply:
yeah. this was pre-existing, but i agree it's not right. we don't track the summarizer anyway, so it will never happen. removing